### PR TITLE
(refactor) Refactor modals to match naming conventions

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR refactors modal files to match the `*.modal.tsx` naming convention introduced in https://github.com/denniskigen/o3-docs/pull/42 and other places.

### CODE REVIEW

File: packages/esm-appointments-app/src/appointments/common-components/end-appointment.test.tsx
Comment (ibacher) [inline packages/esm-appointments-app/src/appointments/common-components/end-appointment.test.tsx:1]: Shouldn't this be named `end-appointment.modal.test.tsx`?

**Review by ibacher — APPROVED** [review]
Code looks good, but I think we should standardize tests having a similar convention to the components they are tests for.

**Comment by denniskigen** [comment]
We don't do component.test.tsx for test files, right? It's typically just the filename (sans the extension) ending with .test.tsx. Same for scss files (sorry about the lack of formatting - typed on my phone).

**Comment by ibacher** [comment]
@denniskigen Yeah, that's normally how we'd do it. I'm a little concerned about what happens when you have a component and, e.g., a modal that embeds that component. I.e., I'd rather let `component.tsx` -> `.test.tsx` be the norm and everything else be namespaced, but maybe that's an unlikely scenario?

**Comment by denniskigen** [comment]
Yeah, I'd imagine that's unlikely. I'll merge this and review the situation post-merge once all the commits are in.

**Pull request merged**